nrespipe/utils.py

```diff
@@ -578,12 +603,17 @@ def get_mag_from_simbad(target_name : str):
 
```

Comment:
It might pay to add some retry logic in here. Your call, but it's saved BANZAI a few times when the archive was having problems. 

https://github.com/LCOGT/banzai/blob/485a13362448a768a41f92c1a56d860d8469064a/banzai/utils/fits_utils.py#L219